docs/Installation.md

```diff
@@ -1,58 +1,135 @@
+# Installation
+
 T4 has two different components:
-* A Python package, `t4`, distributed via `pip`
-* A web interface, the T4 Catalog, deployed using [AWS CloudFormation](https://aws.amazon.com/cloudformation/)
+* A Python package, `t4`
+* A web interface, the T4 Catalog
 
 If you have an already-provisioned catalog, only the Python package is needed.
 
+## T4 client (Python 3.6)
```

Comment:
We should keep the title clean and state that you need Python >=3.6 somewhere in the text, not in the heading.